esp/scm/ws_machine.ecm

```diff
@@ -32,7 +32,6 @@ ESPStruct [nil_remove] DiskUsage
     int64 Available;
     int PercentAvailable;
     [min_ver("1.17")] string Exception;
-    [min_ver("1.18")] int64 ActualSize;
```

Comment:
Feels like removing a versioned variable may give compatibility issues, no?

Reply:
Without this change, the DiskUsage may not contain the ActualSize variable if ESP cannot set the value. So, the variable may not always be there. I think that removing this variable should not give compatibility issues. If really needed, I can add the code to increase the version and depr this variable. Please let me know.